### PR TITLE
Add and refine unittests for VPacket

### PR DESCRIPTION
**Pull request by WTer**

### CODE REVIEW

File: p2p/vpacket.go [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
  */
 package p2p
 
-import "encoding/binary"
+import  (
+    "encoding/binary"
+)
 
 const (
     //VPacketHeadLen head length in net-package
```
Comment (herove) [inline p2p/vpacket.go]: Add a space at the beginning, pay attention to others

Comment (WTer) [reply p2p/vpacket.go]: nice suggestion. fixed it.

Comment (qiaozhigang) [inline p2p/vpacket.go]: dataLen don't need send to peer, because in the receiver end it can be calculated by udp package size and VPacketHeadLen.

Comment (WTer) [reply p2p/vpacket.go]: sure, I already reverted to original version.

**Pull request merged**